docs/source/extension/extension_migration.rst

```diff
@@ -46,19 +46,34 @@ and is based on `FAST <https://www.fast.design/>` library by Microsoft.
 
```

Comment:
The links above do not render properly, seems the syntax is wrong?

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/da47e7d6-9ba8-46f4-b3f3-a343939d50c6)

Reply:
Thanks @krassowski, I updated it